README.md

```diff
@@ -1 +1,46 @@
-# My-Portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My Portfolio
+
+> The Portfolio project is my personal Portfolio, basically the range of my investments throught my training in Microverse.  
+
+![screenshot](./portfolio.PNG)
+
+This project is build in two sections which are the "Header" and the "Headline". The header section uncludes the logo, named "My Logo" and navbar, the second section includes the background template and icons. All elements (background and icons, logo) were exported from Figma phototype.
+
+## Built With
+
+- HTML
+- CSS
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
+- You can clone this repo by typing git clone 'https://github.com/OLIPLICHE/My-Portfolio.git' on your terminal
+- Type cd 'My-Portfolio' to access the project on the terminal
+
+
+### Prerequisites
+- Text editor
+- Git
+- Html
+- Css
+
+👤 **paka**
```

Comment:
- [ ] OPTIONAL
Kindly add a heading `### Author` above your name to ensure a better-structured ReadMe.